controllers/urlShortner.js

```diff
@@ -0,0 +1,65 @@
+const { nanoid } = require('nanoid')
+const urlQuery = require('../models/urls')
+
+const ERROR_MESSAGE = 'Error getting urls'
+const NOTVALIDURL = 'Please attach valid URL'
+const NOTVALIDSHORTURL = 'Please attach valid ID'
+const validUrlRegex = /^(?:(?:(?:https?|ftp):)?\/\/)(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,})))(?::\d{2,5})?(?:[/?#]\S*)?$/i
```

Comment:
7:23  warning  Unsafe Regular Expression  security/detect-unsafe-regex
Currently receiving this warning 
I tried to resolve this issue but have not found the one which is resolving this warning
An alternative would be to install valid-url package which I did found would be overload which could be resolved with above statement

Open to suggestions :)